pkg/services/alerting/engine.go

```diff
@@ -86,32 +87,79 @@ func (e *Engine) runJobDispatcher(grafanaCtx context.Context) error {
 		case <-grafanaCtx.Done():
 			return dispatcherGroup.Wait()
 		case job := <-e.execQueue:
-			dispatcherGroup.Go(func() error { return e.processJob(alertCtx, job) })
+			dispatcherGroup.Go(func() error { return e.processJobWithRetry(alertCtx, job) })
 		}
 	}
 }
 
 var (
 	unfinishedWorkTimeout time.Duration = time.Second * 5
 	alertTimeout          time.Duration = time.Second * 30
+	alertMaxAttempts      int           = 2
```

Comment:
3 ? (plus add TODO to make that configurable in the config file)

Reply:
Done